doas.c

```diff
@@ -416,8 +416,8 @@ main(int argc, char **argv)
 
 	/* setusercontext set path for the next process, so reset it for us */
 	if (rule->cmd) {
-		if (setenv("PATH", safepath, 1) == -1)
```

Comment:
This should not be changed, this is the PATH/environment that is used to lookup the the commands for explicit rules, which should still use the `safepath` search path, otherwise the behavior changes.

Reply:
Ah, my bad, will revert this.